### PR TITLE
perf: add sample code and readme for add sso in c# tab projects

### PR DESCRIPTION
**Pull request by KennethBWSong**
Add sso step and sample: https://github.com/KennethBWSong/AddSsoDotnet/pull/1

### CODE REVIEW

**Review by 1yefuwang1 — APPROVED** [review]

File: packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+3. Create: 'Auth/tab'
+   - Reference code and a 'README.txt' file. These files are provided for reference. See below for more information.
+
+Update your code to add SSO
+-------------------------
+As described above, the Teams Toolkit generated some configuration to set up your application for SSO, but you need to update your application business logic to take advantage of the SSO feature as appropriate.
+
+1. Move 'GetUserProfile.razor' file under 'Auth/tab' to 'Components/'.
+   - 'GetUserProfile': This file implements a function that uses TeamsFx SDK to call Microsoft Graph API to get user info.
+2. Replace the following line: '<AddSSO />' with '<GetUserProfile />' to replace the 'AddSSO' component with 'GetUserProfile' component.
```
Comment (blackchoey) [inline packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt:24]: Tell user which file to edit in this step?

File: packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+-------------------------
+When you added the SSO feature to your application, Teams Toolkit updated your project to support SSO:
+After you successfully added SSO into your project, Teams Toolkit will create and modify some files that helps you implement SSO feature.
+1. Create: 'aad.template.json' under 'templates/appPackage'
+   - The Azure Active Directory application manifest that is used to register the application with AAD.
+2. Modify: 'manifest.template.json' under 'templates/appPackage'
+   - An 'webApplicationInfo' object will be added into your Teams app manifest template. This field is required by Teams when enabling SSO.
+3. Create: 'Auth/tab'
+   - Reference code and a 'README.txt' file. These files are provided for reference. See below for more information.
+
```
Comment (blackchoey) [inline packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt:17]: Will we update appsettings.json for user?

File: packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+-------------------------
+You can debug your application by:
+
+1. Right-click your project and select Teams Toolkit > Prepare Teams app dependencies
+2. If prompted, sign in with an M365 account for the Teams organization you want 
+to install the app to
+3. Press F5, or select the Debug > Start Debugging menu in Visual Studio
+4. In the launched browser, select the Add button to load the app in Teams
+
+Teams Toolkit will use the AAD manifest file to register a AAD application registered for SSO.
```
Comment (blackchoey) [inline packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt:36]: Seems this line is not part of `Debug your application`? Maybe we can to move to `Customize AAD application` section?

File: packages/fx-core/tests/plugins/solution/solution.usertask.test.ts [hunk 4]
```diff
@@ -1124,6 +1124,49 @@ describe("V2 implementation", () => {
...
+        ).activeResourcePlugins.includes(aadPluginV2.name)
+      ).to.be.true;
+      expect(
+        (mockedCtx.projectSetting.solutionSettings as AzureSolutionSettings).capabilities.includes(
+          TabSsoItem.id
+        )
+      ).to.be.true;
+      const readmePath = path.join(testFolder, "Auth", "tab", "README.txt");
+      const readmeExists = await fs.pathExists(readmePath);
+      expect(readmeExists).to.be.true;
```
Comment (blackchoey) [inline packages/fx-core/tests/plugins/solution/solution.usertask.test.ts:1167]: Also check `GetUserProfile.razor` exists?

File: packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt (hunk 1 above)
Comment (KennethBWSong) [reply packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt]: Updated, thx!

File: packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt (hunk 2 above)
Comment (KennethBWSong) [reply packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt:19]: Added logic and readme in the new commit, thx!

File: packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt (hunk 3 above)
Comment (KennethBWSong) [reply packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt:38]: This part exists in `Debug your application` in readme for js and ts, so I prefer to keep the same part here.

File: packages/fx-core/tests/plugins/solution/solution.usertask.test.ts (hunk 4 above)
Comment (KennethBWSong) [reply packages/fx-core/tests/plugins/solution/solution.usertask.test.ts:1178]: Updated, thx!

**Review by blackchoey — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

**Pull request merged**